04-store-api/starter/controllers/products.js

```diff
@@ -0,0 +1,77 @@
+const Product = require("../models/product");
+
+const getAllProductsStatic = async (req, res) => {
+  const products = await Product.find({ price: { $gt: 30 } })
+    .sort("price")
+    .select("name price");
+  // .limit(10)
+  // .skip(1);
+
+  // throw new Error('testing async errors')
+  res.status(200).json({ products, nbHits: products.length });
+};
+
+const getAllProducts = async (req, res) => {
+  const { featured, company, name, sort, fields, numericFilters } = req.query;
+  const queryObject = {};
+
+  if (featured) {
+    queryObject.featured = featured === "true" ? true : false;
```

Comment:
This could be simplified to:
`queryObject.featured = featured === "true"`
since it evaluates to `true` or `false` already.